common/util.go

```diff
@@ -88,6 +88,10 @@ const (
 	taskCompleterMaxInterval        = 10 * time.Second
 	taskCompleterExpirationInterval = 5 * time.Minute
 
+	domainCacheInitialInterval    = 1 * time.Second
+	domainCacheMaxInterval        = 20 * time.Second
```

Comment:
Can I request we make this lower, like 5 seconds I think? I think there's the problem that this logic is smeared around in a few places (at least https://github.com/cadence-workflow/cadence/blob/e36c8185f34a59e321c29db21fcb64a7716be4bd/common/cache/domainCache.go#L63) and relied upon to make decisions such as https://github.com/cadence-workflow/cadence/blob/e36c8185f34a59e321c29db21fcb64a7716be4bd/service/history/task/task.go#L311

I agree it's the subject of unnecessary load, but the domain cache (in)consistency across hosts during transfer tasks is a subject of some pretty difficult-to-figure out bugs

Reply:
Sure - that's also a good justification for looking into this more. Since in the end, there is no guarantee that this will succeed in any reasonable time when it has already started failing.